src/test/kotlin/racingcar/CarTest.kt

```diff
@@ -0,0 +1,35 @@
+package racingcar
+
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.assertAll
+import org.junit.jupiter.params.ParameterizedTest
+import org.junit.jupiter.params.provider.ValueSource
+
+internal class CarTest {
+    private val dummyEngine = object : Engine {
+        override fun run() = Torque(0)
+    }
+
+    @Test
+    fun `차의 초기 위치는 0이다`() {
+        assertThat(Car(dummyEngine).location).isEqualTo(CarLocation(0L))
```

Comment:
아래의 코드로 개선해 보면 어떨까요?

```suggestion
        assertThat(Car(dummyEngine).location).isEqualTo(CarLocation.ZERO)
```